week-2/F-logical-operators/exercise.js

```diff
@@ -12,25 +12,41 @@ var cssLevel = 4;
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
 var htmlLevelAbove5;
+if (htmlLevel > 5) {
+  htmlLevelAbove5 = true;
+} else {
+  htmlLevelAbove5 = false;
+}
 var cssLevelAbove5;
+if (cssLevel > 5) {
+  cssLevelAbove5 = true;
+} else {
+  cssLevelAbove5 = false;
+}
 
-// Finish the next two statement
+// Finish the next  two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
 var cssAndHtmlAbove5;
+if (cssLevelAbove5 === true && htmlLevelAbove5 === true) {
+  cssAndHtmlAbove5 = true;
+} else {
+  cssAndHtmlAbove5 = false;
+}
```

Comment:
Can you do the same thing without using an `if/else` statement? 😛

Reply:
Good use of logical operator